newsletter-february-2023.rst

```diff
@@ -0,0 +1,119 @@
+.. post:: Feb 1, 2023
+   :tags: newsletter, python
+   :author: Ben
+   :location: MLM
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from the last month,
+      current focus, and upcoming features.
+
+Read the Docs newsletter - February 2023
+========================================
+
+News and updates
+----------------
+
+Here are the latest updates from our team since the :doc:`previous newsletter </newsletter-january-2023>`:
+
+- 🪄️ Build outputs are now stored in a well-known location, available to the build process.
+  This opens up many new and exciting possibilities for generating and processing final output formats,
+  which we will uncover in an upcoming blog post.
+  PDFs for MkDocs and encrypted documentations are just two demos that we have ready.
+  **Stay tuned!**
+- 🌪️ Uploads are faster and done in parallel,
+  thanks to `rclone`_. 
+- 🔒️ CORS headers have been tightened on all platforms.
+- 🛠️ The default branch of a git repository is now correctly detected in manual imports.
+  This was a long-standing bug that we are very happy to finally have fixed.
+- 🎤️ Write the Docs has announced their 2023 conferences.
+  Read more in `their announcement`_.
+- 🔒️ Security issue found and fixed: `Path traversal: access to files from any project <GHSA-5w8m-r7jm-mhp9>`_
+- 🔒️ Security issue found and fixed: `Symlink following: Arbitrary file access from builder server <GHSA-hqwg-gjqw-h5wg>`_
+- 🔒️ Security issue found and fixed: `Cache poisoning <GHSA-7fcx-wwr3-99jv>`_
+- 🚦️ Note from Cloudflare that provides CDN for everything on readthedocs.io:
+
+    On February 14th, 2023,
+    Cloudflare will be doing database maintenance that will impact SSL API availability and may result in certificate issuance delays.
+    The scheduled maintenance will be on February 14, 2023, 14:00 - 16:00 UTC.
+
+.. _rclone: https://rclone.org/
+.. _their announcement: https://www.writethedocs.org/blog/2023-january-update/
+.. _GHSA-5w8m-r7jm-mhp9: https://github.com/readthedocs/readthedocs.org/security/advisories/GHSA-5w8m-r7jm-mhp9
+.. _GHSA-hqwg-gjqw-h5wg: https://github.com/readthedocs/readthedocs.org/security/advisories/GHSA-hqwg-gjqw-h5wg
+.. _GHSA-7fcx-wwr3-99jv: https://github.com/readthedocs/readthedocs.org/security/advisories/GHSA-7fcx-wwr3-99jv
+
+You can always see the latest changes to our platforms in our :doc:`Read the Docs Changelog <readthedocs:changelog>`.
+
+
+.. raw:: html
+
+   <a href="https://about.readthedocs.com/" style="background-color: #409cff; border-radius: 3px; color: #ffffff; display: block; margin: 30px auto; font-size: 18px; font-weight: 700; line-height: 24px; padding: 15px 0 15px 0; text-align: center; text-decoration: none; width: 238px;">
+     Discover all the features ✨️
+   </a>
+
+Upcoming features
+-----------------
+
+..
+  Notes:
+
+  Next newsletter:
+  Make a general announcement of our Roadmap
```

Comment:
@ericholscher is there a basis for this? perhaps we'll just push this for March? :)

Reply:
Can you clarify a bit more? This is presumably about our roadmap being publicly accessible for users. 

https://github.com/orgs/readthedocs/projects/156/views/1

It's not super useful tho necessarily :)

Reply:
Yes, that's what it's about. Perhaps it was only on call with @humitos - but we talked about an old intention to make the roadmap not _just_ public but also _announced_, so we would need to motivate users about why to look at our roadmap and how they can use it etc.

It's probably better that we go over this in conversation to see if there's a way forward for this.

Reply:
Yeah, my idea was just to mention that we have a public roadmap in case people want to take a look at it --without going too fancy on this. Just a note, basically.

Reply:
https://github.com/readthedocs/meta/issues/13#issuecomment-1109759481